### ISSUE TITLE
Odd issue with Mongoose middleware

### ISSUE THREAD

**Issue by cyberwombat** [petkaantonov/bluebird#496]
I wanted to bring your attention to this thread. http://stackoverflow.com/questions/28490709/mongoose-promisify-and-middleware?noredirect=1#comment45304575_28490709

Something in Bluebird came and went it seems. Not sure what but thought I would have you take a peek at the post in case it was of use. I got it to work but I still don't know what dis/appeared in 2.9.7.

**Comment by zj8487** [comment]
@cyberwombat  please kindly update to 2.9.8

```
2.9.8 (2015-02-10)

Bugfixes:

Fix regression introduced in 2.9.7 where promisify didn't properly dynamically look up methods on this
```

**Comment by cyberwombat** [comment]
Thank you.
